### PR TITLE
Add recipes for atomium and its dependencies (rmsd, valerius)

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jaimergp** [comment]
@samirelanduk, recipes as per issue https://github.com/samirelanduk/atomium/issues/21 have been added here. Please confirm whether you are alright with being a maintainer of `atomium` and `valerius`!

**Comment by samirelanduk** [comment]
Absolutely! Let me know if there's anything I need to do.

**Comment by jaimergp** [comment]
Why is Python 2.7 being used to build the `noarch` packages? I can't skip it with `build: True` and I guess I shouldn't provide a `conda_build_config.yaml` to override the default `python: [2.7, 3.6, 3.7]` selectors?

**Comment by jaimergp** [comment]
Ok @conda-forge/staged-recipes, this is ready for review!

File: recipes/valerius/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "valerius" %}
+{% set version = "0.2" %}
```
Comment (npavlovikj) [inline recipes/valerius/meta.yaml]: Isn't this package for suitable for bioconda instead, https://bioconda.github.io?

Comment (scopatz) [reply recipes/valerius/meta.yaml]: We generall don't mind having it here though either

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**